model/events/LtiAgsListener.php

```diff
@@ -178,6 +178,7 @@ private function queueSendAgsScoreTaskWithScores(
                 'gradingProgress' => $gradingStatus,
                 'scoreGiven' => $scoreTotal,
                 'scoreMaximum' => $scoreTotalMax,
+                'timestamp' => time(),
```

Comment:
Could you confirm this with Julien, is it ok to send timestamp of moment when we send it instead of e.g. moment when API has been called?